### ISSUE TITLE
Use a particular Rust version

### ISSUE THREAD

**Issue by marmistrz** [rust-lang/rls-vscode#769]
As far as I understand, currently it's impossible to use an older version of Rust with RLS.

I have a [project](https://github.com/golemfactory/sp-wasm) which still requires an older version of Rust. The project is built with 
```
rustup run 1.38.0 cargo build
```
Setting `channel` to `1.38.0` in `settings.json`
```
    "rust-client.channel": "1.38.0",
```
doesn't work and 1.38.0 is not used.

**Comment by Xanewok** [comment]
This worked last time I used it; maybe it didn't refresh properly after setting the value again? Did you try opening the project again with "1.38.0" set from the start?

**Comment by marmistrz** [comment]
Hmmm, now this works for me. I had to manually install `rust-analysis rust-src rls` for that particuar toolchain because of
```
Error: the task 'Rust: Installing `rust-analysis`' neither specifies a command nor a dependsOn property. The task will be ignored. Its definition is:
{
    "type": "shell",
    "id": "shell,rustup component add rust-analysis --toolchain 1.38.0,",
    "problemMatcher": [],
    "label": "Rust: Installing `rust-analysis`"
}
```
I don't know why this didn't work, maybe because I just restarted RLS from within VS Code and not restarted whole VS Code. Anyway, I think that entering "1.38.0" into `settings.json` should not print a "value not accepted" warning in the config editor.

**Comment by Xanewok** [comment]
> Anyway, I think that entering "1.38.0" into settings.json should not print a "value not accepted" warning in the config editor.

This has been fixed in 0.7.2: https://github.com/rust-lang/rls-vscode/commit/3676797e37f7208628bab9fcbb626514f197221c
